config/routes.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
-  root controller: :categories, action: :index
+  root controller: :cats, action: :index
+  resources :categories
```

Comment:
Please scope the routes to be created with `only:`

Reply:
done